client/src/config.js

```diff
@@ -0,0 +1,9 @@
+var PORT = 5000;
+var URL = "http://localhost:5000";
+
+if (process.env.NODE_ENV === "production") {
+    PORT = 80;
+    URL = "http://secretadmirer.me";
+} 
+
+export default [PORT, URL];
```

Comment:
can;t you just return an object so that you don't access it by `[1]` kinda indexing?